lib/tasks/data.rake

```diff
@@ -0,0 +1,65 @@
+namespace :data do
+  require 'open-uri'
+  require 'json'
+
+  def generate_shortname(name)
+    known_exceptions = {
+      "Wellington (A)" => "wellington_nsw",
+      "Kingston (DC)" => "kingston_sa",
+      "Latrobe (M)" => "latrobe_tas",
+      "Murray (A)" => "murray_nsw",
+      "Flinders (S)" => "flinders_qld",
+      "Central Highlands (M)" => "central_highlands_tas"
+    }
+
+    known_exceptions[name] || name.split(" (").first.downcase.gsub(" ", "_")
+  end
+
+  path = File.dirname(__FILE__) + "/../../tmp/lga_2015_aust.geojson"
```

Comment:
If you add in the spatial coordinates, it does get quite large - we've got a few tickets for "show the boundary of the LGA" - so in memory might not be the best. Tempfile; can do.

Why it's done like this - an explicit path to allow you to (re)load without a network delay. Probably more helpful in development that the once or twice you'd run it  in prod.